cmd/server/files.go

```diff
@@ -232,7 +232,7 @@ func getFileContents(logger log.Logger, repo WireFileRepository) http.HandlerFun
 		logger.Log("rendering file contents")
 
 		w.Header().Set("Content-Type", "text/plain")
-		if err := wire.NewWriter(w).Write(file); err != nil {
+		if err := wire.GetWriter(w, r).Write(file); err != nil {
```

Comment:
I'd prefer to keep anything `http` related out of `writer.go`. Can you move `GetWriter()` into this file instead?

Reply:
moved GetWriter() to files.go